### PR TITLE
Removed genemodder positive and negative traits

### PR DESCRIPTION
**Pull request by Jackofashepard**
As a measure to stop the rampant abuse of genemodders and FBP for powergaming i have removed these two trait lists. Perhaps one day in the future i will make a new trait system but for now the neutral traits remain along with rending claws

### CODE REVIEW

**Comment by Jackofashepard** [comment]
image

File: modular_mithra/code/modules/mithra/07_traits.dm [hunk 1]
```diff
@@ -190,42 +146,20 @@ datum/preferences/copy_to(mob/living/carbon/human/character, is_preview_copy = F
 			pref.blood_color = "#f5e400"
 		return TOPIC_REFRESH
 
-	else if(href_list["clicked_pos_trait"])
-		var/datum/trait/trait = text2path(href_list["clicked_pos_trait"])
-		var/choice = alert("Remove [initial(trait.name)] and regain [initial(trait.cost)] points?","Remove Trait","Remove","Cancel")
```
Comment (io-util) [inline modular_mithra/code/modules/mithra/07_traits.dm:195]: You're removing a `text2path` here.

File: modular_mithra/code/modules/mithra/07_traits.dm [hunk 2]
```diff
@@ -190,42 +146,20 @@ datum/preferences/copy_to(mob/living/carbon/human/character, is_preview_copy = F
...
 	else if(href_list["clicked_neu_trait"])
 		var/datum/trait/trait = text2path(href_list["clicked_neu_trait"])
 		var/choice = alert("Remove [initial(trait.name)]?","Remove Trait","Remove","Cancel")
 		if(choice == "Remove")
 			pref.neu_traits -= trait
 		return TOPIC_REFRESH
 
-	else if(href_list["clicked_neg_trait"])
-		var/datum/trait/trait = text2path(href_list["clicked_neg_trait"])
-		var/choice = alert("Remove [initial(trait.name)] and lose [initial(trait.cost)] points?","Remove Trait","Remove","Cancel")
```
Comment (io-util) [inline modular_mithra/code/modules/mithra/07_traits.dm:209]: And here a `text2path` is removed too.

**Review by io-util — CHANGES_REQUESTED** [review]
You removed `text2path` 2 times, as specified in `/test/check-paths.sh` it expects exactly 28, which is not the case since you removed 2 and it's 26 at the moment, this is purely a code quality check, please change `exactly 28 "text2path uses" 'text2path'` at line 36 in `/test/check-paths.sh` to `exactly 26 "text2path uses" 'text2path'`.

**Review by io-util — APPROVED** [review]
Works for me.

**Pull request merged**